### ISSUE TITLE
2.3.0: Missing TweenTransparency extenders for GUIControl inherited structs (Button, Label, etc.)

### ISSUE THREAD

**Issue by edmundito** [edmundito/ags-tween#29]
While `TweenTransparency` for GUIControls was implemented in #18, AGS does not pass down the extender from the GUIControl down to its class inheritance.

Current:
```
import int TweenTransparency(GUIControl* control, ...);
```

Expected:
```
import int TweenTransparency(Button* button, ...);
import int TweenTransparency(Label* button, ...);
// etc.
```

**Comment by ericoporto** [comment]
I made a topic to discuss this in the forums here

https://www.adventuregamestudio.co.uk/forums/index.php?topic=62130.0

It turns out this is possibly just an autocomplete bug, I will look into it and update here.

**Comment by edmundito** [comment]
@ericoporto indeed. From what I've been told it works, it's just that it doesn't appear in autocomplete, so this is a workaround to make sure it appears.

**Comment by ericoporto** [comment]
I believe it should be possible to fix it for 3.6.2.x and 4.0.0.x and forward
